symplyphysics/laws/dynamics/resonant_oscillations.py

```diff
@@ -9,15 +9,17 @@
     validate_input,
     validate_output,
 )
+from symplyphysics.core.expr_comparisons import expr_equals
 from symplyphysics.core.symbols.quantities import scale_factor
+from symplyphysics.laws.dynamics import forced_oscillations_equation as forced_eqn
 
 # Description
 ## When an oscillating external force is driving the oscillations of an oscillator,
 ## the amplitude of oscillations is greatest when the angular frequency of the driving
 ## force is equal to the natural frequency of the oscillator. This condition is called
 ## resonance.
 
-# Law: q(t) = f_m * (w0 * t * sin(w0 * t + phi) - sin(phi) * sin(w0*t)) / (2 * m * w0**2)
+# Law: q(t) = (f_m / m) * t * sin(w0 * t + phi) / (2 * w0)
```

Comment:
I've explained it in the commit description, by the way. Basically, the previous solution accounted for zero initial conditions, but that's not necessary since I only want to describe the particular solution of the differential equation that is related to the oscillator's reaction to the driving force. The $`sin(\varphi) sin(\omega t)`$ factor would yield $`0`$ anyway.